internal/manager/config.go

```diff
@@ -32,12 +32,13 @@ type Config struct {
 	LogReduceRedundancy bool
 
 	// Kong high-level controller manager configurations
-	KongAdminAPIConfig adminapi.HTTPClientOpts
-	KongAdminToken     string
-	KongWorkspace      string
-	AnonymousReports   bool
-	EnableReverseSync  bool
-	SyncPeriod         time.Duration
+	KongAdminAPIConfig  adminapi.HTTPClientOpts
+	KongAdminAPIRetries uint
```

Comment:
This is the new parameter. I am thinking it could have a better name as the rest of the params in the group are the Kong Admin API client settings. KongAdminAPIRetries is more of a KIC setting